desktop/use-desktop/builds.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Explore Builds (Beta)
+description: Understand how to use the Builds view in Docker Desktop
+keywords: Docker Dashboard, manage, gui, dashboard, builders, builds
+---
+
+The **Builds**  view is a simple interface that lets you inspect your build
+history and manage builders using a graphical interface. By default, it
+displays a list of all your ongoing and completed builds. 
+
+> **Beta**
+>
+> The **Builds** view is currently in [Beta](../../release-lifecycle.md/#beta).
+> This feature may change or be removed from future releases.
+
+The **Builds** view displays metadata about the build, such as:
+
+- Build name
+- Target platforms
+- Builder name
+- Build duration
+- Progress bar
+- Cache usage
+- Completion date
+
+The **Active builds** section displays builds that are currently running on
+builders that you're using. The **Completed builds** section lists build
+records for past builds, and includes successful, failed, and stopped builds.
```

Comment:
Maaaybe we should clarify here that it won't list any builds from stopped builders

Reply:
yep, can add that